galleries/olof/style.css

```diff
@@ -0,0 +1,107 @@
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+}
+
+.container {
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 100px repeat(3, 1fr);
+    grid-template-rows: repeat(4, 1fr) 80px;
+    grid-template-areas: 
+    "aside main main main"
+    "aside main main main"
+    "aside main main main"
+    "aside main main main"
+    "footer footer footer footer";
+}
+h1 {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+a {
+    text-decoration: none;
+}
+a:hover {
+    color: red;
+}
+.aside, .main, .footer {
+    border: 3px solid black;
+    margin: 3px;
+}
+.aside {
+    grid-area: aside;
+    display: flex;
+    flex-flow: column;
+    justify-content: space-around;
+    align-items: center;
+}
+.main {
+    padding: 15px;
+    grid-area: main;
+
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    overflow: auto;
+}
+.footer {
+    grid-area: footer;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.box {
+    border: 2px solid black;
+    padding: 10px;
+    margin-bottom: 30px;
+    box-shadow: 2px 2px 2px 2px cornflowerblue;
+}
+.date {
+    color: rgba(0, 0, 0, 0.6);
+}
+.like1 {
+    display: flex;
+    flex-flow: row;
+}
+.like2:hover {
+    color: red;
+    cursor: pointer;
+    opacity: .4;
+}
+.like2 {
+    margin-right: 3px;
+    color: red;
+}
+.boll {
+    display: flex;
+    flex-flow: column;
+}
+.bolltext {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    text-align: center;
+}
+.mainall {
```

Comment:
confusing class name. Since it is only one style and you are adding somewhere else. 

Looking closer at your index.html it looks like it not not being applied at all. Try to remove it.